tracing-example/pom.xml

```diff
@@ -39,7 +39,8 @@
         <version.org.junit>5.10.1</version.org.junit>
         <version.org.wildfly.arquillian>5.0.1.Final</version.org.wildfly.arquillian>
         <version.wildfly-maven-plugin>4.2.0.Final</version.wildfly-maven-plugin>
```

Comment:
The version of the plugin is 5.0.0.Alpha2.

Reply:
Okay!